lang/php8/Makefile

```
@@ -16,7 +16,7 @@ PKG_CPE_ID:=cpe:/a:php:php
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=http://www.php.net/distributions/
```

```suggestion
PKG_SOURCE_URL:=https://www.php.net/distributions/
```

You could probably update this at the same time, it is easily checked by opening https://www.php.net/distributions/php-8.3.3.tar.x/ in your browser

---

Will do as part of new PR.